src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -369,7 +369,7 @@ private async Task CaptureAsyncExceptions(Task task)
         }
     }
 
-    private static partial class Log
+    private static new partial class Log
```

Comment:
Is this `new` really needed here?

Reply:
It's a warning now after I added the project reference.
It is hiding the parent logger. 

A) If we want to use the parent logger, I can delete this one.
B) or I can #pragma the warning